pipeline/e2e-test-publish-results.yaml

```diff
@@ -8,10 +9,3 @@ steps:
       condition: succeededOrFailed()
       displayName: publish e2e test results
       timeoutInMinutes: 3
-
-    - publish: '$(System.DefaultWorkingDirectory)/test-results/e2e'
```

Comment:
This gets pulled in from `e2e-job-per-environment.yaml`. Have we considered the impact of removing these lines in that context?

Reply:
Initially removed this from here and directly included it in the job itself as template Context. However, now I have included this part as a task in this file itself.